src/styles/blocks/product.scss

```diff
@@ -0,0 +1,97 @@
+.product {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
```

Comment:
I'm tried to use box-sizing:border-box, and reduce padding on 1px for to obtain the dimensions required in the layout